core/services/beacon/main.py

```diff
@@ -202,6 +222,44 @@ async def stop(self) -> None:
         await asyncio.gather(*[runner.unregister_services() for runner in self.runners.values()])
 
 
+app = FastAPI(
+    title="Beacon API",
+    description="Beacon is responsible for publishing mDNS domains.",
+    default_response_class=PrettyJSONResponse,
+    debug=True,
+)
+
+beacon = Beacon()
+
+
+@app.get("/services", response_model=List[MdnsEntry], summary="Current domains broadcasted.")
+@version(1, 0)
+def get_services() -> Any:
+    return list(itertools.chain.from_iterable([runner.get_services() for runner in beacon.runners.values()]))
+
+
+@app.get("/get_ip", response_model=IpInfo, summary="Ip Information")
```

Comment:
since this is already a get method, the get prefix in the api name is really necessary ?
As example, services as `get_services` name but the api endpoint is only `services`. We should avoid using get_ prefix for api calls.

Reply:
I'll add a post-it to improve out api design name convention.